packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import Select from 'muicss/lib/react/select'
+import { BaseInput } from '../util/Input'
+
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { useControlRef } from '../util/ControlRef'
+
+export const Kansas = () => {
+  const idRef = useControlRef<Select>()
+  const enrichValues = (baseInfo: StatelessInfo): NoSignature<KansasInfo> | null => {
+    const idData = ''
+    const idType = 'Number'
+
+    return {
+      ...baseInfo,
+      state: 'Kansas',
+      idType,
+      idData,
+    }
+  }
+
+return <SignatureBase<KansasInfo>enrichValues={enrichValues} >
+  Current Kansas driver&apos;s license number or nondriver&apos;s identification card number:
+  <BaseInput
```

Comment:
The PDF actually says "If I do not have a current and valid Kansas driver's license number or Kansas...I must provide a copy of one of the following forms of photo identification..."

So I've asked ludanin@ to make a 'toggalable option' component so that we can show an Input component or Upload component based on whether the user chooses 'License/ID card' or 'I don't have one of those'.

Reply:
To be more clear, this component would be a dropdown (react Option component) that toggles certain children based on the value selected in the dropdown.